nilearn/_utils/param_validation.py

```diff
@@ -0,0 +1,65 @@
+"""
+Utilities to check for valid parameters
+"""
+
+import numbers
+import warnings
+
+from .compat import _basestring
+
+
+def check_threshold(threshold, data, percentile_calculate, name='threshold'):
+    """ Checks if the given threshold is in correct format and within the limit.
+
+    If necessary, this function also returns score of the data calculated based
+    upon the given specific percentile function.
+    Note: This is only for threshold as string.
+
+    Parameters
+    ----------
+    threshold: a float value or a real number or a percentage in string.
+        If threshold is a float value, it should be within the range of the
+        maximum intensity value of the data.
+        If threshold is a percentage expressed in a string it must finish with a
+        percent sign like "99.7%" or just a real number as 99.
+    data: ndarray
+        an array of the input masked data.
+    percentile_calculate: a percentile function {scoreatpercentile, fastabspercentile}
```

Comment:
Just say function on this line and then below something like 

"Percentile function for example scipy.stats.scoreatpercentile"

Also maybe, call the parameter percentile_func which seems slightly better than percentile_calculate.